src/test/resources/config.properties

```diff
@@ -4,4 +4,11 @@ url=https://cubecartuat.unitedcoderapp.com/admin_d9nchr.php
 
 #CREDENTIALS
 username=testautomation1
-password=automationadmin!123
\ No newline at end of file
+password=automationadmin!123
+
+databaseUrl=92.205.12.67
+databasePort=3306
+databaseUsername=ucautomation2023
+databasePassword=automationadmin!123
+databaseSchema=i9362596_cc1
+standAloneDatabase=cubecartstudent2023
```

Comment:
please hide the config file by putting it in the .gitignore

Reply:
It should be in place now.